javascript/javascript1/week2/student-manager.js

```diff
@@ -0,0 +1,36 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  // You write code here
+
+  if (class07Students.length > 6) {
+    console.log("Cannot add more students to class 07");
+  } else if (class07Students.length > 6 && studentName === "Queen") {
+    console.log("Queen added as exeption");
+  } else if (studentName === " ") {
+    console.log("You are not adding any students");
+  }
+
+  for (i = 0; i <= class07Students.length; i++) {
+    if (class07Students[i] === studentName) {
```

Comment:
Here a better condition will be to use *includes*. 
check the usage of includes [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

Reply:
Also I'll suggest you use [Prettier code formatter](https://prettier.io/docs/en/editors.html) extension for VScode to format your code. It makes it cleaner and easier to read.

Reply:
I am already using Prettier, weird :D Fixing right now to includes.

Reply:
Just committed this one too, have some problems with empty string, it is adding it anyway, how to fix it?

Reply:
yes, you can have an if statement to check if studentname is an empty string then you return a warning, also use trim to remove whitspaces from the studentnames so that it'll macth your conditions especially with the "Queen". So in that case "Queen   "  and  "   Queen" will be trimmed to "Queen". 
[Check trim here](https://www.w3schools.com/jsref/jsref_trim_string.asp)